src/core/components/fields/LabeledPasswordField.tsx

```diff
@@ -19,41 +19,45 @@ export interface LabeledPassWordFieldProps extends PropsWithoutRef<JSX.Intrinsic
 
 export const LabeledPasswordField = forwardRef<HTMLInputElement, LabeledPassWordFieldProps>(
   ({ onEyeClick, name, label, outerProps, fieldProps, labelProps, ...props }, ref) => {
+    let validValue = (v) => (v === "" ? null : v)
+    let myType = props.type === "number" ? (Number as any) : validValue
```

Comment:
Similar to #329, it is not obvious as a reviewer why this change is happening. I guess that it has to do with the transform error you mention, but similarly, I don't know what the error is because it has not been documented anywhere.